Add issue to Fleet troubleshooting page for ECE 2.10 / Fleet 7.14 issue (#968)
elastic/observability-docs: docs/en/ingest-management/troubleshooting.asciidoc

// Add Javascript and CSS for tabbed panels
include::{tab-widgets}/code.asciidoc[]

[discrete]
[[fleet-ece-config]]
== {fleet} is not working when enabled in an {ece} deployment

> When an {ece} deployment is set up with APM and {fleet}, even though the deployment operates as expected {fleet} cannot be configured. Agent log files may contain a message like the following:

Reviewer: ```suggestion
When an {ece} deployment is set up with APM and {fleet}, even though the deployment operates as expected, {fleet} cannot be configured. Agent log files may contain a message like the following:
```

[source,sh]
----
WARN [transport] transport/tcp.go:52 DNS lookup failure "instance-000000 0000": lookup instance-0000000000 on 169.254.165.225:23: no such host
----

> This is a known issue with the {stack} pack version 7.14 shipped with {ece} version 2.10. To resolve the problem, link:{ece-ref}/ece-manage-elastic-stack.html[re-upload a fresh copy] of the version 7.14 {stack} pack to overwrite the original one. This issue will also be addressed in later {stack} packs and {ece} versions.

Author: ```suggestion
This is a known issue with the {stack} pack version 7.14 and {ece} version 2.10. With the original {stack} pack version 7.14, if downloaded from the Elastic website before August 10, 2021, Fleet does not work when enabled in a deployment. 

To resolve the problem, link:{ece-ref}/ece-manage-elastic-stack.html[get and re-upload a fresh copy] of the version 7.14 Elastic {stack} pack to overwrite the original one. If you have existing version 7.14 deployments, then restart Fleet/APM after re-uploading the Elastic {stack} pack to enable Fleet. This issue will be addressed in later {stack} packs and {s} versions.
```

> This is a known issue with the Elastic Stack pack version 7.14 shipped with {ece} version 2.10. To resolve the problem, link:{ece-ref}/ece-manage-elastic-stack.html[re-upload a fresh copy] of the version 7.14 Elastic Stack pack to overwrite the original one. This issue will also be addressed in later Elastic stack packs and {ece} versions.

Reviewer: ```suggestion
This is a known issue with the {stack} pack version 7.14 shipped with {ece} version 2.10. To resolve the problem, link:{ece-ref}/ece-manage-elastic-stack.html[re-upload a fresh copy] of the version 7.14 {stack} pack to overwrite the original one. This issue will also be addressed in later {stack} packs and {ece} versions.
```

Reviewer: I'm assuming here that "pack" is an ECE-related term because we don't use "pack" in the stack docs (AFAIK).

Author: Yup, that's exactly right. ECE comes with "stack pack" archived bundles of the stack products.
